app/src/main/res/layout/activity_new_vehicle.xml

```diff
@@ -0,0 +1,150 @@
+                android:layout_below="@id/textView_vehicleModel"
+                android:id="@+id/editText_vehicleModel"
+                android:hint="e.g. Rio, Fiesta, Charger"/>
+
+            <TextView
+                android:text="Color:"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/editText_vehicleModel"
+                android:id="@+id/textView_vehicleColor" />
+
+            <EditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:ems="10"
+                android:inputType="textCapSentences"
+                android:layout_below="@id/textView_vehicleColor"
+                android:id="@+id/editText_vehicleColor"
+                android:hint="e.g. red, blue, green" />
+
+            <TextView
+                android:text="Year:"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/editText_vehicleColor"
+                android:id="@+id/textView_vehicleYear" />
+
+            <EditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:ems="10"
+                android:layout_below="@id/textView_vehicleYear"
+                android:id="@+id/editText_vehicleYear"
+                android:hint="e.g. 2003, 2008, 2013"
+                android:inputType="number" />
+
+            <TextView
+                android:text="Other Notes:"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_below="@id/editText_vehicleYear"
+                android:id="@+id/textView_vehicleOther" />
+
+            <EditText
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:ems="10"
+                android:inputType="textCapSentences"
+                android:id="@+id/editText_vehicleOther"
+                android:hint="e.g. Look for the Alberta sticker!"
+                android:layout_below="@+id/textView_vehicleOther" />
+
+            <Button
+                android:text="Register"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:id="@+id/button_createNewUser"
+                android:layout_below="@id/editText_vehicleOther"
+                android:onClick="register"
+                android:layout_marginTop="10dp"/>
+        </RelativeLayout>
+
+    </ScrollView>
+
+    <View
+        android:id="@+id/bottomDivider"
+        android:layout_below="@+id/scrollView"
+        android:layout_width="match_parent"
+        android:layout_height="2dp"
+        android:background="@color/colorPrimaryLighter" />
+
+
+
+
```

Comment:
So it's gonna be forever
Or it's gonna go down in flames
But I've got a blank space baby
And please for the love of God let me live and delete these lines.